src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -229,6 +229,34 @@ MailQueueMessage CreateMessage(string email, object model)
         await EnqueueMailAsync(messageModels);
     }
 
+    public async Task SendOrganizationInitInviteEmailAsync(string organizationName, OrganizationUser orgUser,
```

Comment:
Is there any reason to keep the separate method, or can we just add `InitOrganization` as a param to the existing method?

Reply:
The reason was that before we had also a separate implementation on the client side. Now it makes sense to remove it